deployment/network-operator/README.md

```diff
@@ -344,6 +378,10 @@ parameters.
 
 | Name                                                 | Type   | Default                                  | Description                                                                                                                                                 |
 |------------------------------------------------------|--------|------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `admissionController.enabled`                        | bool   | `False`                                  | deploy with admission controller webhook                                                                                                                    |
+| `admissionController.useCertManager`                 | bool   | `False`                                  | use cert-manager for generating self-signed certificate                                                                                                     |
+| `admissionController.certificate.tlsCrt`             | string | ``                                       | External certificate crt                                                                                                                                    |
```

Comment:
please add in description that tlsCrt and tlsKey are ignored in case useCertManager is true

Reply:
done